### PR TITLE
Week 2 Technigo project

### PR DESCRIPTION
**Pull request by Ajliin**
Week 2 project

Learned about grid and flexbox, worked fine combining both

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -2,21 +2,136 @@
...
+      </section>
 
-    <section class="">
-      <!-- The other news  -->
+     <section class="grid-news">
+     
+      <!-- Grids x 6 -->
+       <!-- Grid1 -->
+      <div class=grid-content>
+        <img class="grid-picture" src="./pic/grid-river.jpg" alt="">
```
Comment (Mattsson717) [inline code/index.html:43]: You could fill in the alt for the picture here.

File: code/index.html [hunk 2]
```diff
@@ -2,21 +2,136 @@
...
+          <h3> This also happened </h3>
+          <p>
+           Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.
+          </p>
+       </div>
+      </div> 
     </section>
-  </body>
+  
+  </main>
```
Comment (Mattsson717) [inline code/index.html:123]: I haven't seen this main tag before. What does it do? :)

File: code/style.css [hunk 3]
```diff
@@ -1,4 +1,231 @@
+/*
```
Comment (Mattsson717) [inline code/style.css:1]: Are these for reference? Or why is it commented out?

File: code/style.css [hunk 4]
```diff
@@ -1,4 +1,231 @@
...
+  background: #F8EDE3;
   color: #fff;
 }
+
+nav{
+  display: none;
+}
+
+h1 {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
```
Comment (Mattsson717) [inline code/style.css:24]: You have the same font family later on in the CSS, maybe it could be under the html tag :)

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,231 @@
...
+  padding: 50px;
+}
+
+.logo-and-text {
+  display: flex;
+  align-items: center;
+  justify-content: flex;
+  font-size: 1em;
+}
+/* --------BIG NEWS FOR MOBILE ------------------*/
```
Comment (Mattsson717) [inline code/style.css:55]: Your code is well commented and its easy to follow :)

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,231 @@
...
+
+.big-news-img {
+ object-fit: contain;
+ 
+}
+.big-news-text {
+  box-sizing: border-box;
+  padding: 20px;
+  justify-content: center;
+  font-size: 2vh;
```
Comment (Mattsson717) [inline code/style.css:73]: What is vh? :)

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,231 @@
...
+    font-size: 8pt;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+  }
+
+/* --------@MEDIA TABLET---------------------------------*/
+
+
+@media  (min-width: 668px) {
+/* HEADER for tablet*/
+  nav {
```
Comment (Mattsson717) [inline code/style.css:150]: Nice work on getting the navbar to disapear in mobile. :)

**Review by Mattsson717 — COMMENTED** [review]
The site looks really nice, responsive on all platform. Great job!!

File: code/index.html (hunk 1 above)
Comment (Ajliin) [reply code/index.html:43]: Thanks I will

**Pull request closed without merging**